internal/cli/kraft/compose/ls/ls.go

```diff
@@ -86,20 +88,45 @@ func (opts *LsOptions) Run(ctx context.Context, args []string) error {
 		return err
 	}
 
+	machineController, err := mplatform.NewMachineV1alpha1ServiceIterator(ctx)
+	if err != nil {
+		return err
+	}
+
+	machines, err := machineController.List(ctx, &machineapi.MachineList{})
+	if err != nil {
+		return err
+	}
+
 	table.AddField("NAME", cs.Bold)
 	table.AddField("STATUS", cs.Bold)
 	table.AddField("COMPOSEFILE", cs.Bold)
 	table.EndRow()
 
 	for _, project := range projects.Items {
-		var status string
-		if len(project.Status.Machines) == 0 {
-			if !opts.ShowAll {
-				continue
+		status := "Stopped"
```

Comment:
Can you [publicly expose this variable](https://github.com/unikraft/kraftkit/blob/staging/internal/cli/kraft/ps/ps.go#L104), [use similar wrapping logic](https://github.com/unikraft/kraftkit/blob/staging/internal/cli/kraft/ps/ps.go#L250-L252) and then pass the status verbatim? Better to use KraftKit's internal machine API state representation since it maps to `kraft ps` and can help with debugging. Thanks!

Reply:
Thanks. This looks better, too!